src/component/mxgraph/renderer/StyleComputer.ts

```diff
@@ -140,8 +141,11 @@ export default class StyleComputer {
         // arbitrarily increase width to relax too small bounds (for instance for reference diagrams from miwg-test-suite)
         styleValues.set(mxgraph.mxConstants.STYLE_LABEL_WIDTH, labelBounds.width + 1);
         // align settings
-        styleValues.set(mxgraph.mxConstants.STYLE_LABEL_POSITION, mxgraph.mxConstants.ALIGN_TOP);
-        styleValues.set(mxgraph.mxConstants.STYLE_VERTICAL_LABEL_POSITION, mxgraph.mxConstants.ALIGN_LEFT);
+        // According to the documentation, "label position" can only take values in left, center, right with default=center
+        // However, there is undocumented behavior when the value is not one of these and this behavior is exactly what we want.
+        // See https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/view/mxGraphView.js#L1183-L1252
+        styleValues.set(mxgraph.mxConstants.STYLE_LABEL_POSITION, 'ignore');
```

Comment:
**question:** we may have used _none_ instead of _ignore_. _none_ is already used in some mxGraph style configuration.

Reply:
To have consistency with other value of the `mxgraph` properties, I prefer `none` value to `ignore`.

Reply:
Fine I am going to handle it

Reply:
done in 185fbb5c

Reply:
reverted in ba368fa. For rationale, see https://github.com/process-analytics/bpmn-visualization-js/pull/2551#issuecomment-1452084806.